dbms/src/Storages/DeltaMerge/ColumnFile/ColumnFileSetReader.h

```diff
@@ -40,10 +40,15 @@ class ColumnFileSetReader
 
     std::vector<ColumnFileReaderPtr> column_file_readers;
 
+    LACBytesCollectorPtr lac_bytes_collector;
+
 private:
     explicit ColumnFileSetReader(const DMContext & context_)
         : context(context_)
-    {}
+    {
+        if (context.scan_context)
+            lac_bytes_collector = std::make_unique<LACBytesCollector>(context.scan_context->resource_group_name);
```

Comment:
maybe `lac_bytes_collector(context.scan_context ？ context.scan_context->resource_group_name : "")` is enough.
But I think it's okay.

Reply:
changed.